### DISCUSSION TITLE
Discord / IRC server??

### DISCUSSION THREAD

**Discussion by bbrendon** [blakeblackshear/frigate#14763] [General]
Since the HASS Discord Server shutdown the chat rooms, is there a Frigate chat room somewhere?

**Comment by chrisuthe** [comment]
Would love to see a dedicated discord, but that is a time sink for anyone willing to help as well, so, tradeoffs. Huge fan of old school forums (like the one we are on right now) for knowledge durability and searchability, but that doesn't mean either is the _right_ way to go about it.
